### ISSUE TITLE
Feature Request: Move ~/.face to ~/.local/share/lightdm/profile to better comply with FreeDesktop Base Directory Specifications

### ISSUE THREAD

**Issue by BeatLink** [canonical/lightdm#133]

**Comment by Redsandro** [comment]
`.face` is used by multiple DMs. Might make sense to use the same file shared by the different DMs a user has installed.

**Comment by BeatLink** [comment]
Thats true. Would it be possible to add a flag that checks for both locations? First the ~/.local/share/lightdm/profile  then if that doesnt exist using ~/.face? That would be the best of both worlds by supporting the base directory spec when possible while also being able to support other DMs
